meta.go

```diff
@@ -238,6 +238,22 @@ func parseStringList(list interface{}) []string {
 	return result
 }
 
+func checkTerm(s string) error {
+	if strings.Contains(s, " ") {
+		return errors.New("term may not contain spaces: " + s)
+	}
+	tokens := strings.Split(s, "/")
+	if len(tokens) != 2 {
+		return errors.New("unknown term format: " + s)
+	}
+
+	_, err := strconv.Atoi(tokens[1])
```

Comment:
I think `tokens[0]` ought to match a regexp like `^[a-z]+([a-z0-9-]+)?$`:
- Must start with a lowercase letter
- Followed by any combination of lowercase letters a-z, digits and dashes